common/flatpak-run.c

```diff
@@ -2475,7 +2475,7 @@ setup_seccomp (FlatpakBwrap   *bwrap,
     {SCMP_SYS (clone), &SCMP_A0 (SCMP_CMP_MASKED_EQ, CLONE_NEWUSER, CLONE_NEWUSER)},
 
     /* Don't allow faking input to the controlling tty (CVE-2017-5226) */
-    {SCMP_SYS (ioctl), &SCMP_A1 (SCMP_CMP_EQ, (int) TIOCSTI)},
+    {SCMP_SYS (ioctl), &SCMP_A1 (SCMP_CMP_MASKED_EQ, ~0u, (int) TIOCSTI)},
```

Comment:
If what you want is to match "low 32 bits are TIOCSTI", would a mask of `0xFFFFFFFFu` be more obviously correct? (You need quite a high level of C expertise to know how many bits are set in `~0u`.)

Reply:
> You need quite a high level of C expertise to know how many bits are set in `~0u`

Somewhat debatable, but changed anyway :)